### PR TITLE
Some language fixes

### PR DESCRIPTION
**Pull request by OzancanKaratas**
- Compliance for spelling rules.
- Compliance with each other the strings.

May I add the iOS strings before merging?

### CODE REVIEW

File: res/values/strings.xml [hunk 1]
```diff
@@ -26,7 +26,7 @@
     <string name="hide_password">Hide password</string>
     <string name="not_now">Not now</string>
     <string name="never">Never</string>
-    <string name="one_moment">One moment…</string>
+    <string name="one_moment">One moment...</string>
```
Comment (r10s) [inline res/values/strings.xml]: "…" is the unicode character 'HORIZONTAL ELLIPSIS'; following most recommendations and style guides, this should be favored over the three dots afaik.

File: res/values/strings.xml [hunk 2]
```diff
@@ -441,17 +441,17 @@
     <string name="pref_watch_inbox_folder">Watch Inbox folder</string>
     <string name="pref_watch_sent_folder">Watch Sent folder</string>
     <string name="pref_watch_mvbox_folder">Watch DeltaChat folder</string>
-    <string name="pref_auto_folder_moves">Automatic Moves to DeltaChat folder</string>
+    <string name="pref_auto_folder_moves">Automatic moves to DeltaChat folder</string>
```
Comment (r10s) [inline res/values/strings.xml]: thanks for pointing that out, i corrected this in another commit.

**Comment by OzancanKaratas** [comment]
@Jikstra I want to discus with you. I want to help and language problems is very important for me. This pull request corrects the problems I noticed.

File: res/values/strings.xml [hunk 3]
```diff
@@ -441,17 +441,17 @@
...
     <string name="pref_auto_folder_moves_explain">Chat conversations are moved to avoid cluttering the Inbox folder</string>
     <string name="pref_email_interaction_title">Email interaction</string>
     <string name="pref_show_emails">Show classic emails</string>
     <string name="pref_show_emails_no">No, chats only</string>
     <string name="pref_show_emails_accepted_contacts">For accepted contacts</string>
     <string name="pref_show_emails_all">All</string>
     <string name="pref_empty_server_title">DANGER ZONE: Delete e-mails from server</string>
-    <string name="pref_empty_server_msg">This function helps to free space on your IMAP server by deleting ALL e-mails, including chat-messages, in the given folders. Messages on this device will not be deleted.  The deletion cannot be undone!</string>
-    <string name="pref_empty_server_inbox">Delete ALL e-mails in Inbox folder</string>
-    <string name="pref_empty_server_mvbox">Delete ALL e-mails in DeltaChat folder</string>
```
Comment (r10s) [inline res/values/strings.xml]: i think the intention of the upper-case ALL is some emphasis. but i agree that this is a bit odd as it is not used somewhere else. changing this in another pr.

File: res/values/strings.xml [hunk 4]
```diff
@@ -556,10 +556,10 @@
...
+    <string name="perm_explain_access_to_camera_denied">Delta Chat requires the Camera permission in order to take photos or videos, but it has been permanently denied. Please go to the app settings menu, select \"Permissions\", and enable \"Camera\".</string>
+    <string name="perm_explain_access_to_mic_denied">Delta Chat requires the Microphone permission in order to send audio messages, but it has been permanently denied. Please go to app settings menu, select \"Permissions\", and enable \"Microphone\".</string>
+    <string name="perm_explain_access_to_storage_denied">Delta Chat requires the Storage permission in order to attach or export photos, videos, or audio, but it has been permanently denied. Please go to the app settings menu, select \"Permissions\", and enable \"Storage\".</string>
+    <string name="perm_explain_access_to_location_denied">Delta Chat requires Location permission in order to attach a location, but it has been permanently denied. Please go to the app settings menu, select \"Permissions\", and enable \"Location\".</string>
     <string name="perm_enable_bg_reminder_title">Tap here to receive messages while Delta Chat is in background.</string>
```
Comment (r10s) [inline res/values/strings.xml]: not sure if "go to" is better than "continue to" here, my english is not good enough to decide that. btw, the wording is not from delta chat but from the signal source - who should at least native speakers, if that means anything :)

**Comment by OzancanKaratas** [comment]
@r10s I think you should handle each commits individually.

**Comment by r10s** [comment]
ups, i've just noticed there are more commit, i will go to them as well.

wrt keys: we cannot change them easily as they're also used on ios, windows, mac etc. and changing them produces a lot of follow-up work that is usually not worth the effort.

File: res/values/strings.xml [hunk 5]
```diff
@@ -305,7 +306,7 @@
     <string name="welcome_intro6_message">Encrypted with all common algorithms. Messages stay on your servers.</string>
     <string name="welcome_intro7_headline">Trustworthy</string>
     <string name="welcome_intro7_message">Delta Chat is safe for business use, compatible and standards-based.</string>
-    <string name="welcome_start_messaging">Start Messaging</string>
+    <string name="welcome_start_messaging">Start messaging</string>
```
Comment (r10s) [inline res/values/strings.xml]: i correct that in another pr.

File: src/org/thoughtcrime/securesms/permissions/Permissions.java [hunk 6]
```diff
@@ -156,8 +156,8 @@ private void executePreGrantedPermissionsRequest(PermissionsRequest request) {
     @SuppressWarnings("ConstantConditions")
     private void executePermissionsRequestWithRationale(PermissionsRequest request) {
       RationaleDialog.createFor(permissionObject.getContext(), rationaleDialogMessage, rationalDialogHeader)
-                     .setPositiveButton(R.string.perm_continue, (dialog, which) -> executePermissionsRequest(request))
-                     .setNegativeButton(R.string.perm_not_now, (dialog, which) -> executeNoPermissionsRequest(request))
+                     .setPositiveButton(R.string.continue, (dialog, which) -> executePermissionsRequest(request))
+                     .setNegativeButton(R.string.not_now, (dialog, which) -> executeNoPermissionsRequest(request))
```
Comment (r10s) [inline src/org/thoughtcrime/securesms/permissions/Permissions.java]: i will remove that double string, thanks for the hint!

**Comment by r10s** [comment]
> May I add the iOS strings before merging?

ios just uses the same strings as android, nothing more to do :)

**Pull request closed without merging**